pathd/path_pcep_controller.c

```diff
@@ -756,7 +920,162 @@ int pcep_main_event_handler(struct thread *thread)
 
 
 /* ------------ Helper functions ------------ */
+bool is_best_pce(struct ctrl_state *ctrl_state, int pce)
+{
+	if (ctrl_state && ctrl_state->pcc[pce]) {
```

Comment:
This shouldn't be stored there. Two variable in the controller struct that point to the best and maybe the previous best would work fine AFAIK.